python/taichi/lang/kernel.py

```diff
@@ -87,19 +87,8 @@ def do_compile(self):
         func_body = tree.body[0]
         func_body.decorator_list = []
 
-        if impl.get_runtime().print_preprocessed:
-            import astor
-            print('Before preprocessing:')
-            print(astor.to_source(tree.body[0], indent_with='  '))
-
         visitor = ASTTransformer(is_kernel=False, func=self)
         visitor.visit(tree)
-        ast.fix_missing_locations(tree)
```

Comment:
Is it safe to remove this line? Maybe we need this for the correct line number information. I'm not sure.

Reply:
It's already included in `ASTTransformer.visit`, so no worry.